src/main/java/io/cdap/plugin/gcp/datastore/sink/DatastoreSinkConfig.java

```diff
@@ -179,7 +179,8 @@ public boolean shouldUseAutoGeneratedKey() {
   }
 
   public void validate(@Nullable Schema schema) {
-    super.validate();
+    // TODO: (vinisha) add failure collector
```

Comment:
will this be addressed immediately after this?

Reply:
yes